h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/exec/RemoteXGBoostExecutorTest.java

```diff
@@ -91,4 +93,34 @@ public void prostateRegressionWithCustomEarlyStopping() {
         }
     }
 
+    @Test
+    public void prostateRegressionWithValidationFrameAndEvalMetric() {
+        Scope.enter();
+        try {
+            // Parse frame into H2O
+            Frame fr = Scope.track(parseTestFile("./smalldata/prostate/prostate.csv"));
+
+            // We want to hit all nodes to confirm distributed XGBoost works
+            fr = ensureDistributed(fr);
+
+            Frame[] splits = SplitFrame.splitFrame(fr, 0.8, 0.2);
+            Scope.track(splits);
+            
+            XGBoostModel.XGBoostParameters parms = new XGBoostModel.XGBoostParameters();
+            parms._train = splits[0]._key;
+            parms._valid = splits[1]._key;
+            parms._response_column = "AGE";
+            parms._ignored_columns = new String[]{"ID"};
+            parms._ntrees = 30;
+            parms._eval_metric = "rmse";
+            parms._score_tree_interval = 3;
+            parms._score_eval_metric_only = true;
+
+            XGBoostModel model = new XGBoost(parms).trainModel().get();
+            Scope.track_generic(model);
```

Comment:
Can we have some assert here? I have run it on master and it still passing, so I am not sure if it is testing anything.

Reply:
Yes, very good point it is missing the check!

Reply:
Fixed, thanks again!